requirements.txt

```
@@ -1,3 +1,4 @@
+pynacl
```

maybe same here as well
```suggestion
pynacl>=1.4.0
```

---

agreed!